gradle-plugin/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'groovy'
+    id 'java-gradle-plugin'
+    id 'maven-publish'
+    id 'com.palantir.git-version' version '0.10.1'
+}
+
+group = "com.lightbend.akka.grpc"
+version versionDetails().gitHashFull.substring(0, 8)
```

Comment:
:+1: for now at least :+1:

Reply:
Yea. We might wanna tag `v0.0.1` soon just to be able to find all the places that need to be changed.